components/rss/actions/merge-rss-feeds/merge-rss-feeds.js

```diff
@@ -0,0 +1,94 @@
+const Parser = require('rss-parser');
+
+module.exports = {
+  name: "Merge RSS Feeds",
+  description: "Retrieve multiple RSS feeds and return a merged array of items sorted by date.",
+  key: "rss-merge-rss-feeds",
+  version: "0.2.1",
```

Comment:
Can you adjust to v `0.0.1`? Currently we fix the first version of the source to that string, and follow standard semver from there (bug fixes get minor bump, added functionality gets minor bump, etc.)